### ISSUE TITLE
"Use this display" option does not have an effect / does nothing.

### ISSUE THREAD

**Issue by murermader** [elementary/switchboard-plug-display#217]
### Description

The "Use this display" settings is ignored for all my monitors. After clicking apply, the monitors appear greyed out, but nothing has changed.

If you need some other debugging information, tell me where I can find it. This is my first bug report, so I hope the information here is somewhat helpful.

#### `ddcutil capabilities` output

```
Failure getting EDID for /dev/i2c-2: status code=DDCRC_EDID(-3016): invalid EDID
MCCS version: 2.2
Commands:
   Command: 01 (VCP Request)
   Command: 02 (VCP Response)
   Command: 03 (VCP Set)
   Command: 07 (Timing Request)
   Command: 0c (Save Settings)
   Command: e3 (Capabilities Reply)
   Command: f3 (Capabilities Request)
VCP Features:
   Feature: 04 (Restore factory defaults)
   Feature: 10 (Brightness)
   Feature: 12 (Contrast)
   Feature: 14 (Select color preset)
      Values:
         05: 6500 K
         06: 7500 K
         08: 9300 K
         0b: User 1
   Feature: 16 (Video gain: Red)
   Feature: 18 (Video gain: Green)
   Feature: 1A (Video gain: Blue)
   Feature: 59 (6 axis saturation: Red)
   Feature: 5A (6 axis saturation: Yellow)
   Feature: 5B (6 axis saturation: Green)
   Feature: 5C (6 axis saturation: Cyan)
   Feature: 5D (6 axis saturation: Blue)
   Feature: 5E (6 axis saturation: Magenta)
   Feature: 60 (Input Source)
      Values:
         0f: DisplayPort-1
         11: HDMI-1
         12: HDMI-2
   Feature: 62 (Audio speaker volume)
   Feature: 9B (6 axis hue control: Red)
   Feature: 9C (6 axis hue control: Yellow)
   Feature: 9D (6 axis hue control: Green)
   Feature: 9E (6 axis hue control: Cyan)
   Feature: 9F (6 axis hue control: Blue)
   Feature: A0 (6 axis hue control: Magenta)
   Feature: D6 (Power mode)
   Feature: E0 (manufacturer specific feature)
      sValues: 00 04 05 06 (interpretation unavailable)
   Feature: E1 (manufacturer specific feature)
      sValues: 00 01 02 (interpretation unavailable)
   Feature: E2 (manufacturer specific feature)
      sValues: 00 01 02 03 05 06 07 0b 10 11 12 (interpretation unavailable)
   Feature: E3 (manufacturer specific feature)
   Feature: E4 (manufacturer specific feature)
   Feature: E5 (manufacturer specific feature)
   Feature: E7 (manufacturer specific feature)
      sValues: 00 01 02 (interpretation unavailable)
   Feature: E8 (manufacturer specific feature)
      sValues: 00 01 02 03 04 (interpretation unavailable)
```

#### `xandr -q` output

```
Screen 0: minimum 8 x 8, current 3584 x 1440, maximum 32767 x 32767
DVI-D-0 connected 1024x768+2560+0 (normal left inverted right x axis y axis) 380mm x 300mm
   1280x1024     60.02 +  75.02  
   1024x768      75.03*   60.00  
   800x600       75.00    60.32  
   640x480       75.00    59.94  
HDMI-0 disconnected (normal left inverted right x axis y axis)
HDMI-1 disconnected (normal left inverted right x axis y axis)
DP-0 connected primary 2560x1440+0+0 (normal left inverted right x axis y axis) 597mm x 336mm
   2560x1440    144.00*+ 120.00    59.95    59.95  
   1920x1080    119.88    60.00    59.94    50.00  
   1680x1050     59.95  
   1440x900      59.89  
   1280x1024     75.02    60.02  
   1280x960      60.00  
   1280x800      59.81  
   1280x720      60.00    59.94    50.00  
   1152x864      75.00  
   1024x768      75.03    70.07    60.00  
   800x600       75.00    72.19    60.32    56.25  
   720x576       50.00  
   720x480       59.94  
   640x480       75.00    72.81    59.94  
DP-1 disconnected (normal left inverted right x axis y axis)
DP-2 disconnected (normal left inverted right x axis y axis)
DP-3 disconnected (normal left inverted right x axis y axis)
```

**Comment by murermader** [comment]
When you get redirected to the issue via the settings app, there is no template, so I didn't see it. So it guess that is another bug. I will report this again, with the template next time.

**Comment by jeremypw** [comment]
This is a duplicate of #131 and is being addressed by #197
